maxentep/utils.py

```diff
@@ -175,30 +179,187 @@ def merge_history(base, other, prefix=''):
     return base
 
 
-def compartment_restrainer(restrained_patches, restrained_compartments, npoints, ref_traj, prior, noise=0, start_time=None, end_time=None, time_average=7):
-    number_of_restrained_compartments = len(restrained_compartments)
+# def compartment_restrainer(restrained_patches, restrained_compartments, npoints, ref_traj, prior, noise=0, start_time=None, end_time=None, time_average=7):
```

Comment:
Can we just remove this? We'll have the commented out source in commit history

Reply:
removed